### PR TITLE
[aptos-cli] Fix output of voting

### PR DESCRIPTION
**Pull request by gregnazario**
### Description
Fixes the output of voting, and removes an unused input field for creating proposals.

### Test Plan
E2E tests

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3429)

### CODE REVIEW

File: crates/aptos/src/move_tool/mod.rs [hunk 1]
```diff
@@ -159,26 +155,14 @@ pub fn init_move_dir(
             .as_path(),
     )?;
 
-    let mut dependencies = BTreeMap::new();
```
Comment (movekevin) [inline crates/aptos/src/move_tool/mod.rs:162]: Are we switching back to local dependencies? I thought we wanted to use the fullnode address from the profile?

File: crates/aptos/src/move_tool/mod.rs [hunk 2]
```diff
@@ -137,7 +135,6 @@ impl CliCommand<()> for InitPackage {
         init_move_dir(
             package_dir.as_path(),
             &self.name,
-            "devnet",
```
Comment (movekevin) [inline crates/aptos/src/move_tool/mod.rs:140]: What was this for? Why remove now?

File: crates/aptos/src/governance/mod.rs [hunk 3]
```diff
@@ -475,6 +467,6 @@ impl CliCommand<&'static str> for GenerateUpgradeProposal {
         let package = BuiltPackage::build(package_path, options)?;
         let release = ReleasePackage::new(package)?;
         release.generate_script_proposal(account, output)?;
-        Ok("success")
+        Ok(())
```
Comment (movekevin) [inline crates/aptos/src/governance/mod.rs:470]: Why?

File: crates/aptos/src/move_tool/mod.rs (hunk 1 above)
Comment (gregnazario) [reply crates/aptos/src/move_tool/mod.rs:162]: Yeah.. this doesn't seem to work yet, so went back to the framework-rev

File: crates/aptos/src/governance/mod.rs (hunk 3 above)
Comment (gregnazario) [reply crates/aptos/src/governance/mod.rs:476]: This was following non-standard outputs.  I've fixed it accordingly

**Review by sherry-x — APPROVED** [review]

**Pull request merged**